src/i18n/locales/en.json

```diff
@@ -509,7 +509,8 @@
     "show_in_dependency_graph": "Show in dependency graph",
     "show_complete_graph": "Show complete graph",
     "not_found_in_dependency_graph": "Dependency could not be found in dependency graph",
-    "reindex": "Rebuild index(es)"
+    "reindex": "Rebuild index(es)",
+    "show_update_information": "Show update information"
```

Comment:
> @rbt-mm For the sake of better readability, the version status icon will only be shown for outdated components.

Given this constraint, I think this should say "highlight outdated components" or so. "Show update information" is rather broad, we should be more specific here.